### DISCUSSION TITLE
Can´t archive Ebay-Auctions

### DISCUSSION THREAD

**Discussion by TobiasHonscha** [ArchiveBox/ArchiveBox#1483] [Q&A]
Example: https://www.ebay.de/itm/266922776037
The auction is archived but one can´t access all the images (try to click on it!) of the item. Maybe not all of the javascript code is downloaded? I use the wget archive ...

**Comment by THonscha** [comment]
I still have the problem to archive ebay auctions:
Example : [https://www.ebay.de/itm/355894734783](https://www.ebay.de/itm/355894734783)

**Comment by pirate** [comment]
Wget does not run any JS, so it's likely to break on JS-heavy sites like `ebay`.

I recommend using the singlefile, dom, and screenshot, methods.

You can also try the latest BETA release with many improvements and fixes: `archivebox/archivebox:dev`.
